src/core_modules/capture-core/components/D2Form/asyncHandlerHOC/actions.js

```diff
@@ -39,10 +38,6 @@ export const fieldsValidated = (
     actionCreator(actionTypes.FIELDS_VALIDATED)(
         { fieldsUI, formBuilderId, formId, validatingUids });
 
-export const cleanUpFormBuilder = (remainingUids: Array<string>, formId: string) =>
-    actionCreator(actionTypes.CLEAN_UP_FORM_BUILDER)(
-        { remainingUids, formId });
-
```

Comment:
I removed this since I didnt find anywhere that is being used. Let me know if you think I missed anything Joakim.